app/api/feeds.py

```diff
@@ -0,0 +1,71 @@
+from app import app
+from flask import jsonify, abort
+from flask.ext.restful import Api, Resource, reqparse, fields, marshal
+
+from datetime import datetime, date
+
+api = Api(app)
+
+feeds = [
+	{
+		'id': 1,
+		'title': u'NYTimes',
+		'url': u'http://rss.nytimes.com/services/xml/rss/nyt/HomePage.xml',
+		'last_updated': datetime.utcnow()
+	},
+	{
+		'id': 2,
+		'title': u'CNN',
+		'url': u'http://rss.cnn.com/rss/cnn_topstories.rss',
+		'last_updated': datetime.utcnow()
+	}
+]
+
+feed_fields = {
+    'title': fields.String,
+    'url': fields.String,
+    'last_updated': fields.DateTime,
+    'uri': fields.Url('feed')
+}
+
+class FeedListAPI(Resource):
+
+	def __init__(self):
+		self.reqparse = reqparse.RequestParser()
+		self.reqparse.add_argument('title', type=str, required=True,
```

Comment:
Is 'title' strictly required or something we should fill in given the URL since we won't be scraping at first?